### ISSUE TITLE
About Training Loss

### ISSUE THREAD

**Issue by CIntellifusion** [End2End-Diffusion/REPA-E#8]
https://github.com/End2End-Diffusion/REPA-E/blob/3ad80468e78ec4fc93e6e70aa08e7b3b8303a171/train_repae.py#L414

Dear authors, thanks for your excellent work. I got a small question about the second repa loss computation. 

I see in this line,  z is detached but the vae is not in the eval mode, so it will still produce gradients but not updated?  Could you explain why the vae is not set to require_grad = False and vae.eval()

**Comment by CIntellifusion** [comment]
Sorry, I think detach() will not produce gradient on vae. So it doesn't need to eval().
